modules/machine-user-infra-machines-pxe.adoc

```diff
@@ -27,18 +27,30 @@ TIMEOUT 20
 PROMPT 0
 LABEL pxeboot
     KERNEL http://<HTTP_server>/rhcos-<version>-installer-live-kernel-<architecture> <1>
-    APPEND initrd=http://<HTTP_server>/rhcos-<version>-installer-live-initramfs.<architecture>.img console=ttyS0 console=tty0 coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://<HTTP_server>/worker.ign coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-installer-live-rootfs.<architecture>.img <2>
+    APPEND initrd=http://<HTTP_server>/rhcos-<version>-installer-live-initramfs.<architecture>.img coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://<HTTP_server>/worker.ign coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-installer-live-rootfs.<architecture>.img <2>
 ----
 <1> Specify the location of the live `kernel` file that you uploaded to your HTTP server.
 <2> Specify locations of the {op-system} files that you uploaded to your HTTP server. The `initrd` parameter value is the location of the live `initramfs` file, the `coreos.inst.ignition_url` parameter value is the location of the worker Ignition config file, and the `coreos.live.rootfs_url` parameter value is the location of the live `rootfs` file. The `coreos.inst.ignition_url` and `coreos.live.rootfs_url` parameters only support HTTP and HTTPS.
++
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```

Comment:
Hmm, this version includes both the original text and my rewrite, which makes for some extra redundancy and a difficult-to-follow subject shift after the second sentence.  I had intended for my text to replace the existing text.  Can we consolidate the two versions?